app/views/public/planning_guides/drawings.html.erb

```diff
@@ -22,8 +22,7 @@
         <li>have a unique drawing reference number</li>
       </ul>
       <p class="govuk-body">
-        Your application may not be marked as valid if the drawings do not clearly
-        and accurately describe the site and the proposed changes.
+        Your application may not be valid if the drawings do not clearly and accurately describe the site and the proposed changes. Upload scans of physical documents rather than photos.
```

Comment:
Cam we extract this string into the translations file and reuse it in all of the places it appears?